modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -968,6 +968,15 @@ definition {
 		}
 	}
 
+	macro validatePredefinedSelectFromList {
```

Comment:
@LucasAcacio, from what I could see from this change, it's duplicating the next macro `validatePredefinedSelectList` instead of renaming it, can you please rename the existing macro instead of duplicate it?

Please update its invocations accordingly.

Thanks. :slightly_smiling_face:

Reply:
Sorry, I misunderstood it before. Proper changes were done

Reply:
No problem. :slightly_smiling_face: